iModelCore/ECPresentation/Source/PresentationManagerImpl.cpp

```diff
@@ -1198,7 +1198,13 @@ ContentDescriptorCPtr RulesDrivenECPresentationManagerImpl::_GetNodesDescriptor(
             ),
         *KeySet::Create(params.GetParentNodeKey() ? NavNodeKeyList{ params.GetParentNodeKey() } : NavNodeKeyList{})
         ), params);
-    return GetContentDescriptor(descriptorParams);
+    auto descriptor = ContentDescriptor::Create(*GetContentDescriptor(descriptorParams));
+    if (descriptor.IsValid())

```

Comment:
`ContentDescriptor::Create` will always return a descriptor, but `GetContentDescriptor(descriptorParams)` is not guaranteed to. This should be changed to:
```cpp
auto descriptor = GetContentDescriptor(descriptorParams);
if (descriptor.IsNull())
    return nullptr;

auto result = ContentDescriptor::Create(*descriptor);
...
```

Reply:
fixed